### PR TITLE
Weather APP

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
### Tools and Techs used
- JavaScript
- HTML and CSS
- Bootstrap
- Eslint

### CODE REVIEW

File: src/functions/apifunc.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      const myTable = document.getElementById('myTable');
+      myTable.innerHTML = '';
+      myTable.appendChild(
+        myResultsTable(
+          response.main.humidity,
+          response.main.pressure,
+          response.main.temp,
+        ),
+      );
+    });
```
Comment (whiz25) [inline src/functions/apifunc.js]: - Wrap this block in a `try/catch` for error handling
- Separate JS logic from DOM manipulation. This file should only contain JS code.

File: src/components/inputForm.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  button.innerText = 'search';
+  button.classList.add('btn', 'btn-primary');
+  button.addEventListener('click', (e) => {
+    e.preventDefault();
+    const cityName = myInput.value;
+    apiCall(cityName);
+    const formHeader = document.getElementById('header');
+    formHeader.innerHTML = '';
+    formHeader.appendChild(myInputForm());
+  });
```
Comment (whiz25) [inline src/components/inputForm.js]: - DOM manipulation logic should live in the `dom` folder.

File: src/components/weatherData.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  const tableSide = document.createElement('div');
+  tableSide.classList.add('text-light',
+    'text-center', 'bg-ifo',
+    'col-6');
+  tableSide.innerHTML = 'Search for any city and get the weather Stats';
+  tableSide.id = 'myTable';
+  fullBodyContent.appendChild(cityName);
+  fullBodyContent.appendChild(tableSide);
+  return fullBodyContent;
```
Comment (whiz25) [inline src/components/weatherData.js]: - Move any DOM logic to the `dom` folder

File: README.md [hunk 4]
```diff
@@ -1 +1,58 @@
...
+- Stylint
+
+## Live Demo
+
+[Live Demo Link](https://emmanuelaaron.github.io/weatherApp/dist/)
+
+
+## Getting Started
+- Clone this repo on your local machine
+- Navigate to this project on your machine
```
Comment (whiz25) [inline README.md:22]: - Give explicit commands to be run

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Good job on this PR.
### Nice work 🌵 🌵 🌵 
- Use of `promises`
- Logic divide into modules
- Can search for weather for a specific location
### Changes to make 🙅🏿 🙅🏿 🙅🏿 
- Add a toggle button to flip the temperature value between Celsius and Fahrenheit
- The background image should change based on the temperature. For example, I should see the sun when it's sunny, etc
- Separate JS logic from DOM manipulation logic
- Check comments for suggestions


### Martin Afani
#### Slack: whiz25

**Review by oloomoses — CHANGES_REQUESTED** [review]
### Changes Required :recycle: 
Hello @Emmanuelaaron,

**You have done a great job implementing the changes suggested by @whiz25. :muscle:**
However, some issues have not been handled properly.

### To improve
- [ ] Kindly Add a button to toggle the temperature value between Celsius and Fahrenheit. You can either use a radion or toggle button
- [ ] The app should change its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png))

**Kindly make the above changes and ask for another review**

Feel free to reach me on slack or leave a comment while mentioning me and I will  be happy to respond

_Slack: Oloo Moses_

**Review by MkrtichSargsyan — APPROVED** [review]
## _Status:_ Approved  :hamster: :rocket:
Hello :wave:.

Awesome job on this project.:rocket: You have completed all the requested changes from the previous code reviewer.

You can go ahead and merge this branch to the base branch of this repo.
Good luck in your future projects and happy coding :relaxed:

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

**Pull request merged**